src/components/Activities/Activity1/Game2/components/GameSlide.tsx

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+
+import useSound from 'use-sound';
+
+import AlienSvg from '../../../../../assets/activity1/game2/alien.svg';
+import CorrectSFX from '../../../../../assets/activity1/game2/correct.mp3';
+import IncorrectSFX from '../../../../../assets/activity1/game2/oh_no_1.mp3';
+
+import { TextBubbleStyles } from '../../../../shared/PlaynetConstants';
+import TextBubble from '../../TextBubble';
+import SlideBox from './SlideBox';
+
+
+interface GameSlideProps {
+  correctImg ?: number;
+  advanceGame ?: () => void;
+  textDefault ?: string;
+  textIncorrect ?: string;
+  imgOption0 ?: string;
+  imgOption1 ?: string;
```

Comment:
```suggestion
  correctImg?: number;
  advanceGame?: () => void;
  textDefault?: string;
  textIncorrect?: string;
  imgs?: string[],
```

and then you can destructure images later

Reply:
you could also always have the first image in `imgs` ALWAYS be the right image..

so `imgs = [correct, incorrect]`

Reply:
Ooo cool idea

Reply:
@tr89on can we add a comment that the first image is `correct` and the second is `incorrect`? this is for easier reuse by others so they don't necessarily have to read all the code to figure out how to use your component!

Reply:
actually nvm just realized you have a `correctImg` prop!